staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -190,71 +186,83 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	case "get":
 		handler := handlers.GetResource(storage, storage, requestScope)
 		handler(w, req)
-		return
 	case "list":
 		forceWatch := false
 		handler := handlers.ListResource(storage, storage, requestScope, forceWatch, minRequestTimeout)
 		handler(w, req)
-		return
 	case "watch":
 		forceWatch := true
 		handler := handlers.ListResource(storage, storage, requestScope, forceWatch, minRequestTimeout)
 		handler(w, req)
-		return
 	case "create":
 		if terminating {
 			http.Error(w, fmt.Sprintf("%v not allowed while CustomResourceDefinition is terminating", requestInfo.Verb), http.StatusMethodNotAllowed)
 			return
 		}
 		handler := handlers.CreateResource(storage, requestScope, discovery.NewUnstructuredObjectTyper(nil), r.admission)
 		handler(w, req)
-		return
 	case "update":
 		if terminating {
 			http.Error(w, fmt.Sprintf("%v not allowed while CustomResourceDefinition is terminating", requestInfo.Verb), http.StatusMethodNotAllowed)
 			return
 		}
 		handler := handlers.UpdateResource(storage, requestScope, discovery.NewUnstructuredObjectTyper(nil), r.admission)
 		handler(w, req)
-		return
 	case "patch":
 		if terminating {
 			http.Error(w, fmt.Sprintf("%v not allowed while CustomResourceDefinition is terminating", requestInfo.Verb), http.StatusMethodNotAllowed)
 			return
 		}
 		handler := handlers.PatchResource(storage, requestScope, r.admission, unstructured.UnstructuredObjectConverter{})
 		handler(w, req)
-		return
 	case "delete":
 		allowsOptions := true
 		handler := handlers.DeleteResource(storage, allowsOptions, requestScope, r.admission)
 		handler(w, req)
-		return
 	case "deletecollection":
 		checkBody := true
 		handler := handlers.DeleteCollection(storage, checkBody, requestScope, r.admission)
 		handler(w, req)
-		return
 
 	default:
 		http.Error(w, fmt.Sprintf("unhandled verb %q", requestInfo.Verb), http.StatusMethodNotAllowed)
+	}
+}
+
+func (r *crdHandler) updateCustomResourceDefinition(oldObj, newObj interface{}) {
```

Comment:
please avoid moving code and changing it. It's hard to review.

Reply:
just move before `remove`. Original  location makes it far way from other  methods of `crdHandler`

Reply:
Github forgot to give me xmas present of detecting those pure code moves.